depthai_nodes/ml/parsers/parser.py

```diff
@@ -1,10 +1,24 @@
-from typing import Dict, List, Union
+from typing import Dict, List, Union, Optional
 
 import depthai as dai
 
 
 class Parser(dai.node.ThreadedHostNode):
```

Comment:
Should we rename?

Reply:
We can rename to `BaseParser` yeah to be inline with other libraries (luxonis-train, luxonis-ml)